tool/site/client/src/pages/claimPage.tsx

```diff
@@ -97,7 +98,8 @@ async function retrievePrescriptionDetails(baseUrl: string, prescriptionId: stri
   const prescriptionDetails: PrescriptionDetails = {
     patient: getPatientResources(prescriptionOrder)[0],
     medicationRequests: getMedicationRequestResources(prescriptionOrder),
-    medicationDispenses: dispenseNotifications.flatMap(getMedicationDispenseResources)
+    medicationDispenses: dispenseNotifications.flatMap(getMedicationDispenseResources),
+    organization: dispenseNotifications.flatMap(getOrganizationResources)[0]
```

Comment:
do we want to maybe be clear that this is the dispensing organization rather than prescribing?

Reply:
done